types/index.d.ts

```diff
@@ -0,0 +1,800 @@
+// Based on definitions by Zach Posten for React-Filepond <https://github.com/zposten>
+// Updated by FilePond Contributors
+
+import * as React from 'react';
+
+// This line shuts off automatic exporting for this module
+// I want to do this so that I can have internal types to this module that make
+// understanding the type definition more clear, while at the same time only
+// exporting the API that react-filepond exports.
+export {};
+
+export enum FileStatus {
```

Comment:
Rather than duplicating [some of these types](https://github.com/pqina/filepond/blob/fae3ade9b8aedb20292eeefebea7c3547264461e/types/index.d.ts#L6) from vanilla  filepond, isn't it possible to import them in a similar way to how you're importing the React types to use here?

My thought is that the benefit of defining these types here instead of [in DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/47008) is that this library has `filepond` as a dependency, so you should have access to the types it already defines.

Reply:
Good idea, thanks. I have pushed an update that does exactly that